clm_data/README.md

```diff
@@ -1,21 +1,107 @@
+  - `medlynintercept(lat, lon)`: Medlyn intercept of conductance-photosynthesis relationship (umol m^-2 s^-1)
+  - `taulnir(lat, lon)`: Leaf transmittance: near-IR (fraction)
+  - `taulvis(lat, lon)`: Leaf transmittance: visible (fraction)
+  - `tausnir(lat, lon)`: Stem transmittance: near-IR (fraction)
+  - `tausvis(lat, lon)`: Stem transmittance: visible (fraction)
+  - `vcmx25(lat, lon)`: Maximum rate of carboxylation (umol CO2/m**2/s)
+
+### 3. `mechanism_map.nc`
+Contains two variables which describe the photosynthesis mechanism for each grid cell.
+`c3_dominant` has a value of 1.0 when C3 is the dominant photosynthesis mechanism for that grid cell and a
+value of 0.0 if C4 is dominant. `c3_proportion` has values in [0.0, 1.0] which represent the proportion of plants
+in a grid cell that use C3 photosynthesis.
+- **Contents**:
+  - `c3_dominant(lat, lon)` (0.0 and 1.0)
+  - `c3_proportion(lat, lon)` (0 to 1)
+
+## Scripts
+
+### 1. `dominant_pft.py`
+- **Description**: Script to determine the dominant PFT for each grid cell.
+- **Purpose**: Processes the surface data to create the `dominant_PFT_map.nc` file.
+
+### 2. `pft_variables.py`
+- **Description**: Main script for mapping vegetation properties based on the dominant PFT.
+- **Functionality**:
+  - Reads the dominant PFT for each grid cell from `dominant_PFT_map.nc`.
+  - Reads physiological parameters from `pft-physiology.c110225.nc`.
+  - Maps these parameters to the global grid based on the dominant PFT.
+  - Outputs the mapped parameters to `vegetation_properties_map.nc`.
+
+### 3. `dominant_mechanism.py`
+- **Description**: Script to determine the dominant photsynthesis mechanism for each grid cell.
+- **Functionality**:
+  - Reads the percent plant functional type for each grid cell (% of total plants in cell) from `surfdata_0.9x1.25_16pfts__CMIP6_simyr2000_c170616.nc`.
+  - If the PFT with the highest percentage in a cell is not PFT #14, then the cell is marked as C3 dominant, and a 1.0 is placed in the grid cell in the output parameter `c3_dominant`. If the highest percentage is PFT #14, a value of 0.0 is placed in the grid cell.
+  - For `c3_proportion`, the value for each cell is the total percentage of the non-C4 PFTs, which is the sum of the percentages for all PFTS except #14.
+  - Outputs the mapped parameters to `mechanism_map.nc`
+- **Purpose**: Processes the surface data to create the `mechanism_map.nc` file.
+
 ## References
 For additional context on the development and capabilities of the Community Land Model, refer to the following publication:
 - Lawrence, D. M., Fisher, R. A., Koven, C. D., Oleson, K. W., Swenson, S. C., Bonan, G., et al. (2019). The Community Land Model Version 5: Description of New Features, Benchmarking, and Impact of Forcing Uncertainty. *Journal of Advances in Modeling Earth Systems*, 11(12), 4245–4287. [DOI:10.1029/2018MS001583](https://doi.org/10.1029/2018MS001583)
```

Comment:
Should there be more references here?

Reply:
No need. This reference is pretty inclusive.